src/ontology/mondo-edit.obo

```diff
@@ -531251,10 +531249,18 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0957421
 name: borna virus encephalitis
-subset: orphanet_rare {source="Orphanet:637051"}
-subset: rare
+def: "A human disease caused by infection with Borna disease virus." [MONDO:patterns/infectious_disease_by_agent]
+comment: In Mondo we make the distinction between human and non-human animal diseases. Cross-references do not necessarily make this distinction. Annotate to the human or non-human animal term as appropriate. This disease is considered zoonotic. {source="PMID:31924550", source="PMID:34376142"}
```

Comment:
- 'comment' should not have a source. Instead, please add the PMID in the text.
- Here is an alternate text:
This disease is considered zoonotic, i.e. able to be transmitted between human and non-human animals (PMID:34376142). In Mondo, we distinguish between human and non-human animal diseases, and we currently do have a general term to represent a species-agnostic version of the disease. This record is for the human version of “borna disease”.

Reply:
Revised comment.